### PR TITLE
km dcp 20 release updates

### PR DESCRIPTION
**Pull request by kayleemathews**

### CODE REVIEW

File: content/community-update/community-update/dcp-updates.md [hunk 1]
```diff
@@ -6,6 +6,35 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### September 26, 2022
+
+### Release Highlights
+
+* We've reached the 20th release of the DCP with data from over 30 million cells! Thank you to the HCA community and all of the contributors that helped us reach this milestone!
```
Comment (ekiernan) [inline content/community-update/community-update/dcp-updates.md]: We typically don't say data release - we could say data portal now hosts data for over 30 million cells.

Comment (kayleemathews) [inline content/community-update/community-update/dcp-updates.md]:
```suggestion
* The Data Portal now hosts over 30 million cells! Thank you to the HCA community and all of the contributors that helped us reach this milestone!
```

**Review by ekiernan — APPROVED** [review]
LGTM! Thank you!

**Comment by ekiernan** [comment]
@NoopDog Please merge this fork. Thank you!

**Review by NoopDog — APPROVED** [review]

**Pull request merged**